actor/src/lib.rs

```diff
@@ -166,6 +168,27 @@ impl<A: Actor> Addr<A> {
         }
     }
 
+    /// Send a message without waiting for an answer.
```

Comment:
@SamHSmith  are you sure that we would send without waiting?
From the docs and source code for [sender](https://docs.rs/tokio/latest/src/tokio/sync/mpsc/bounded.rs.html#654-656) it seems like code will block until receiving an answer.

Reply:
It blocks if there is not space in the message pipe. In the normal case where everything is functioning it puts the message in the pipe and then returns directly.

Reply:
Ok, resolve than